definitions/subworkflows/joint_genotype.cwl

```diff
@@ -0,0 +1,210 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: Workflow
+label: "joint genotyping for trios or small cohorts"
+requirements:
+    - class: ScatterFeatureRequirement
+    - class: MultipleInputFeatureRequirement
+
+    - class: SubworkflowFeatureRequirement
+    - class: SchemaDefRequirement
+      types:
+          - $import: ../types/vep_custom_annotation.yml
+    - class: StepInputExpressionRequirement
+    - class: InlineJavascriptRequirement
+inputs:
+    reference:
+        type:
+            - string
+            - File
+        secondaryFiles: [.fai, ^.dict]
+    gvcfs:
+       type: File[]
+    intervals:
+        type:
+            type: array
+            items:
+                type: array
+                items: string
+    vep_cache_dir:
+        type:
+            - string
+            - Directory
+    vep_ensembl_assembly:
+        type: string
+        doc: "genome assembly to use in vep. Examples: GRCh38 or GRCm38"
+    vep_ensembl_version:
+        type: string
+        doc: "ensembl version - Must be present in the cache directory. Example: 95"
+    vep_ensembl_species:
+        type: string
+        doc: "ensembl species - Must be present in the cache directory. Examples: homo_sapiens or mus_musculus"
+    vep_plugins:
+        type: string[]
+        default: [Downstream, Wildtype]
+    synonyms_file:
+        type: File?
+    annotate_coding_only:
+        type: boolean?
+    vep_custom_annotations:
+        type: ../types/vep_custom_annotation.yml#vep_custom_annotation[]
+        doc: "custom type, check types directory for input format"
+    limit_variant_intervals:
+        type: File
```

Comment:
It wasn't immediately clear to me what the difference between "intervals" and "limit_variant_intervals" was. A doc line on the two intervals inputs wouldn't be an awful idea.   Otherwise, looks good!

Reply:
I've change the `limit_variant_intervals` to `roi_intervals` and added doc lines. 
In a future PR when I add the padding for the target regions I'll use the `roi_intervals` to genotype over those regions instead of limiting after genotyping.